scripts/dev.sh

```
@@ -0,0 +1,319 @@
+      return
+    fi
+  fi
+  echo "an unidentified process"
+}
+
+# Walks forward from base_fastid_port until it finds a free one, printing
+# what it skipped past along the way. Gives up after 5 tries.
+resolve_fastid_port() {
+  local port="$base_fastid_port"
+  local i
+  for i in 1 2 3 4 5; do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)